snapcraft/tests/test_common.py

```diff
@@ -23,6 +23,17 @@
 
 class CommonTestCase(tests.TestCase):
 
+    def setUp(self):
+        super().setUp()
+        self.addCleanup(self._reset_default_paths)
+
+    def _reset_default_paths(self):
```

Comment:
sure, that works too. Inherit from the base test case, and add any paths it could be missing.